calico-enterprise_versioned_docs/version-3.18/releases.json

```diff
@@ -1,259 +1,263 @@
 [
   {
-    "title": "master",
+    "title": "v.3.18.0",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.30.5",
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.26",
+      "minor_version": "v3.25",
```

Comment:
Please confirm with @mgleung  what this version should be. 
This should be coming and/or be in sync with calico-private/node/Makefile: v3.26.0
https://github.com/tigera/calico-private/blob/1b8509021733afab7003d2b8a02c307e014c135d/node/Makefile#L45

Reply:
this is still based on 3.25.1. We have a large PR that I haven't been able to bank yet that will catch everything up to 3.26.1 but it is not in the 3.18 branch.

Reply:
This was resolved per a discussion on slack.